tests/types/cross_module_resolvers/c_mod.py

```diff
@@ -1,13 +1,11 @@
 from typing import List
 
-import a_mod
-import b_mod
-import x_mod
-from a_mod import AObject as C_AObject
-from b_mod import BObject as C_BObject
-
 import strawberry
 
+from . import a_mod, b_mod, x_mod
```

Comment:
I've added a bunch of missing `__init__.py` files, not needed for this PR, but still :)